wiki/docker.md

```diff
@@ -33,91 +32,96 @@ Now docker image has build with tag "eiffel-intelligence-backend:1.0.1"
+Intelligence Spring properties:
 
+[application.properties](../src/main/resources/application.properties)
 
 ### Docker flags
 
+<B>"--expose 8080"</B> - this Docker flag tells that containers internal 
+port shall be exposed to outside of the Docker Host. This flag do not set 
+which port that should be allocated outside Docker Host on the actual server/machine.
 
-<B>"--expose 8080"</B> - this Docker flag tells that containers internal port shall be exposed to outside of the Docker Host. This flag do not set which port that should be allocated outside Docker Host on the actual server/machine.
-
-
-<B>"-p 8070:8080"</B> - this Docker flag is mapping the containers external port 8034 to the internal exposed port 8091. Port 8034 will be allocated outside Docker host and user will be able to access the containers service via port 8034.
-
+<B>"-p 8070:8080"</B> - this Docker flag is mapping the containers external 
+port 8034 to the internal exposed port 8091. Port 8034 will be allocated 
+outside Docker host and user will be able to access the containers service 
+via port 8034.
 
-When Eiffel-Intelligence container is running on your local Docker host, Eiffel-Intelligence should be reachable with address "localhost:8070/\<Rest End-Point\>" or "\<docker host ip\>:8070/\<Rest End-Point\>"
+When Eiffel Intelligence container is running on your local Docker host, 
+Eiffel Intelligence should be reachable with address "localhost:8070/\<Rest End-Point\>" or "\<docker host ip\>:8070/\<Rest End-Point\>"
 
+Another option to configure Eiffel Intelligence is to provide the application 
+properties file into the container, which can be made in two ways:
 
-Another option to configure Eiffel-Intelligence is to provide the application properties file into the container, which can be made in two ways:
-
-**1** Put application.properties file in Tomcat Catalina config folder in container and run Eiffel-Intelligence:
+**1** Put application.properties file in Tomcat Catalina config folder in container and run Eiffel Intelligence:
     
     docker run -p 8070:8080 --expose 8080 --volume /path/to/application.properties:/usr/local/tomcat/config/application.properties eiffel-intelligence:0.0.19
 
-**2** Put application.properties file in a different folder in container and tell EI where the application.properties is located in the container:
-    
-    docker run -p 8070:8080 --expose 8080 --volume /path/to/application.properties:/tmp/application.properties -e spring.config.location=/tmp/application.properties eiffel-intelligence:0.0.19
+**2** Put application.properties file in a different folder in container and 
+tell Eiffel Intelligence where the application.properties is located in the container:
 
+    docker run -p 8070:8080 --expose 8080 --volume /path/to/application.properties:/tmp/application.properties -e spring.config.location=/tmp/application.properties eiffel-intelligence:0.0.19
 
 # Run Docker image with provided docker-compose file
```

Comment:
Mattias commented earlier that titles should be capitilized, does the same apply here?

Reply:
Yes, I'm updating the other headings also :) And this should probably not be a first level heading since it's in the middle of the file.